doc/modules/mixture.rst

```diff
@@ -139,7 +139,8 @@ VBGMM: variational Gaussian mixtures
 ====================================
 
 The :class:`VBGMM` object implements a variant of the Gaussian mixture
-model with :ref:`variational inference <variational_inference>` algorithms. The API is identical to
+model with :ref:`variational inference <variational_inference>` algorithms.
+The API is identical to
 :class:`GMM`. It is essentially a middle-ground between :class:`GMM`
```

Comment:
As VBGMM and DPGMM are not changed yet, I hesitated to remove these lines.
Maybe, it's better to remove it definitely.